### PR TITLE
Create a test suite for kubevirt plugin

### PR DESCRIPTION
**Pull request by irosenzw**
Signed-off-by: Ido Rosenzwig <irosenzw@redhat.com>

### CODE REVIEW

File: frontend/integration-tests/protractor.conf.ts [hunk 1]
```diff
@@ -71,6 +71,9 @@ const testSuites = {
     '../packages/operator-lifecycle-manager/integration-tests/scenarios/global-installmode.scenario.ts',
     '../packages/operator-lifecycle-manager/integration-tests/scenarios/single-installmode.scenario.ts',
   ]),
+  kubevirtPlugin: suite([
```
Comment (rhrazdil) [inline frontend/integration-tests/protractor.conf.ts:74]: we can put this in our plugin kubevirt-plugin package.json, for example as kubevirt-gating

File: test-suites.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```
Comment (rhrazdil) [inline test-suites.sh:1]: This should probably call the test-gui script, shouldn't it?
But do we need to run multiple suites? I thought that the idea was to add a new job running single suite.

Comment (irosenzw) [reply test-suites.sh:1]: we don't run multiple suites - we need to run only kubevirt-gating suite in addition to the other tests that run regularly in each PR.

File: frontend/integration-tests/protractor.conf.ts (hunk 1 above)
Comment (irosenzw) [reply frontend/integration-tests/protractor.conf.ts]: done.

File: test-suites.sh (hunk 2 above)
Comment (irosenzw) [reply test-suites.sh]: done.

**Comment by rhrazdil** [comment]
looks good
/lgtm

**Pull request merged**